### ISSUE TITLE
Notification drawer heading template is wrapped in h4

### ISSUE THREAD

**Issue by benjaminapetersen** [patternfly/angular-patternfly#539]
The current `notification-drawer.html` template wraps the user defined `heading.html` (from `headingInclude`) in an `h4`, which is restrictive for styling beyond a simple heading. 

[Here](https://openshift.github.io/openshift-origin-design/web-console/4-patterns/notifications) is where we are going with the openshift web-console:

<img width="269" alt="screen shot 2017-08-01 at 10 04 26 am" src="https://user-images.githubusercontent.com/280512/28829427-b15e3dfc-76a1-11e7-943e-77b989d462d4.png">

From `notification-drawer.html` it would be ideal to eliminate the `h4`. If preferred for out of the box styling, move it inside the `heading.html` example template:

```html
        <div class="panel-heading">
          <h4 class="panel-title">
            <a ng-if="!$ctrl.singleGroup" ng-click="$ctrl.toggleCollapse(notificationGroup)" ng-class="{collapsed: !notificationGroup.open}" ng-include src="$ctrl.headingInclude"></a>
            <span ng-if="$ctrl.singleGroup" ng-include src="$ctrl.headingInclude"></span>
          </h4>
          <span class="panel-counter" ng-include src="$ctrl.subheadingInclude"></span>
        </div>
```

@jeff-phillips-18 thoughts?

**Comment by jeff-phillips-18** [comment]
we will need to look into backwards compatibility issues that could arise from making this change but I can understand the reasoning.

**Comment by dtaylor113** [comment]
Could we eliminate the `h4` and update `.panel-title` and set font-weight and size to match what an h4 would be?  Then apps could overwrite the `.panel-title` class.

**Comment by jeff-phillips-18** [comment]
We could. The issue would be any applications already overriding this using the h4 as the selector.

**Comment by benjaminapetersen** [comment]
How closely is angular-patternfly trying to follow SemVer?  Strictly no breaking changes until next major release?

**Comment by jeff-phillips-18** [comment]
Yes.

**Comment by benjaminapetersen** [comment]
So Angular (2) seems to follow SemVer, but is on version `4.4.2` with `5.0.0-beta.7 ` coming up soon.  They seem to bump the major version semi-frequently, so the SemVer signal "something breaks" is there, but its not a big delayed rollout.

Curious if that would work for angular-patternfly.  Following SemVer is great if 1.) you can go ahead and release the major as often as you have a breaking change (even if minor like an `h4`), or 2.) if code changes are so infrequent that its ok to wait a relatively long time for a major release. 

I'm guessing its hard to put enough architectural time into each new component to ensure tweaks (breaking tweaks) are avoided.

**Comment by jeff-phillips-18** [comment]
Yes, that is the struggle. I don't see the h4 really causing enough of an issue to warrant making the change and creating a major release. Is this something that you feel can not be (easily) worked around?

**Comment by benjaminapetersen** [comment]
Agree.  Not a problem for now, I think we have an acceptable workaround in place in the console.

**Comment by jeff-phillips-18** [comment]
OK, closing this issue. Feel free to reopen if you'd like it kept in the backlog for the next major release.

**Comment by benjaminapetersen** [comment]
I'd probably prefer to keep it open.  Its definitely still strange to work around the heading.

**Comment by jeff-phillips-18** [comment]
OK, thanks @benjaminapetersen
